zkevm-circuits/src/evm_circuit.rs

```diff
@@ -405,10 +416,10 @@ pub mod test {
                 ]
             })
             .collect();
+        let active_rows = EvmCircuit::get_active_rows(&block);
```

Comment:
I wonder if it's possible to extract the active rows having a `MockProver` instance and if so, I would implement this functionality for `MockProver` instead.

On that way, we are able to test all of our test circuits on that manner, not only this one. And also we don't need this fn call, and instead we can simply use `prover.verify_at_active_rows`.

I belive it should be possible if we return `active_rows` after calling `MockProver::run`.

Reply:
it may not be easy? since selectors like "q_step" are active column, it may be difficult for MockProver to know them as  zero..

Reply:
I see. Makes sense! It's sad but I guess it is what we have for now.